avidemux/qt4/i18n/avidemux_ko.ts

```diff
@@ -945,11 +945,11 @@ Drop timing informations ?</source>
     <message>
         <location line="+10"/>
         <source>© 2001 - 2017  Mean</source>
-        <translation>© 2001 - 2017  Mean</translation>
+        <translation>© 2001 - 2018  Mean</translation>
```

Comment:
Please do not correct strings which are wrong but don't need to be translated via translation. They need to be fixed in the application. The right thing to do would be to replace `<translation>© 2001 - 2017  Mean</translation>` with `<translation></translation>`.

Reply:
If it should be fixed in the application, I have no idea the reason why the string is in translation text though I don't really care about it.

Reply:
The string is picked up automatically while parsing avidemux/qt4/ADM_userInterfaces/ADM_dialog/about.ui where we have a few strings which really need translation, so excluding the entire file is not an option.

Translation files will be regenerated shortly before release and changes to this string in the source will invalidate existing translations.